bug: panic with variadic

```
panic: runtime error: index out of range [4] with length 4

goroutine 7157 [running]:
github.com/ghostiam/protogetter.(*processor).process(0xc000847860, {0x7fa780?, 0xc013978b40})
	/home/ldez/sources/golangci-lint-linters/protogetter/processor.go:111 +0xbbd
github.com/ghostiam/protogetter.Process(0x516199?, 0xc00e7334d0?, {0x7fa780?, 0xc013978b40?}, 0xc026b90990?)
	/home/ldez/sources/golangci-lint-linters/protogetter/processor.go:29 +0x72
github.com/ghostiam/protogetter.analyse(0xc003a78620, 0xc006ddb1b0, {0x7fa780, 0xc013978b40}, 0xc0000adc00)
	/home/ldez/sources/golangci-lint-linters/protogetter/protogetter.go:140 +0x9e
github.com/ghostiam/protogetter.Run.func1({0x7fa780?, 0xc013978b40?})
	/home/ldez/sources/golangci-lint-linters/protogetter/protogetter.go:122 +0x4c
golang.org/x/tools/go/ast/inspector.(*Inspector).Preorder(0xc000847b98, {0xc000847bb0?, 0x3?, 0x75ed27bf58d8?}, 0xc000847c30)
	/home/ldez/sources/go/pkg/mod/golang.org/x/tools@v0.25.0/go/ast/inspector/inspector.go:82 +0x8f
github.com/ghostiam/protogetter.Run(0xc003a78620, 0xc0000adc00)
	/home/ldez/sources/golangci-lint-linters/protogetter/protogetter.go:121 +0x6ca
github.com/ghostiam/protogetter.NewAnalyzer.func1(0x0?)
	/home/ldez/sources/golangci-lint-linters/protogetter/protogetter.go:31 +0x17
golang.org/x/tools/go/analysis/internal/checker.(*action).execOnce(0xc0139d25a0)
	/home/ldez/sources/go/pkg/mod/golang.org/x/tools@v0.25.0/go/analysis/internal/checker/checker.go:759 +0xad0
sync.(*Once).doSlow(0xc00b02f0c0?, 0xc00d585aa0?)
	/usr/lib/go/src/sync/once.go:78 +0xab
sync.(*Once).Do(...)
	/usr/lib/go/src/sync/once.go:69
golang.org/x/tools/go/analysis/internal/checker.(*action).exec(...)
	/home/ldez/sources/go/pkg/mod/golang.org/x/tools@v0.25.0/go/analysis/internal/checker/checker.go:666
golang.org/x/tools/go/analysis/internal/checker.execAll.func1(0x0?)
	/home/ldez/sources/go/pkg/mod/golang.org/x/tools@v0.25.0/go/analysis/internal/checker/checker.go:654 +0x3b
created by golang.org/x/tools/go/analysis/internal/checker.execAll in goroutine 6
	/home/ldez/sources/go/pkg/mod/golang.org/x/tools@v0.25.0/go/analysis/internal/checker/checker.go:660 +0x191

```

```go
package proto

import (
	"net/http"
	"testing"
)

func _(tb testing.TB, client *http.Client, req *http.Request, statusCode int) {
	Errorf(tb, false, "expected %v status code, got %v %v", statusCode, req.URL, client.Transport)
}

func Errorf(tb testing.TB, cond bool, format string, args ...any) {}

```

--- a/processor.go
+++ b/processor.go
@@ -102,6 +102,10 @@ func (c *processor) process(n ast.Node) (*Result, error) {
 				}
 			}
 
+			if len(paramTypes) != len(x.Args) {
+				return &Result{}, nil
+			}
+
 			for i, arg := range x.Args {
 				a, ok := arg.(*ast.SelectorExpr)
 				if !ok {

--- a/testdata/proto/tassert.go
+++ b/testdata/proto/tassert.go
@@ -1,0 +1,12 @@
+package proto
+
+import (
+	"net/http"
+	"testing"
+)
+
+func _(tb testing.TB, client *http.Client, req *http.Request, statusCode int) {
+	Errorf(tb, false, "expected %v status code, got %v %v", statusCode, req.URL, client.Transport)
+}
+
+func Errorf(tb testing.TB, cond bool, format string, args ...any) {}

